molecule/default/requirements.yml

```diff
@@ -0,0 +1,9 @@
+---
+roles:
+  - src: git+https://github.com/infrawatch/functional-tests
+    name: functional-tests
```

Comment:
Is this the #106 violation? AFAICT nothing in the code actually requires the `-`. Would changing this to `functional_tests` (here and L54 of verify.yml) allow us to remove the exception from skip_list?

Reply:
That might have caused it too, but I thought it was because the ansible roles patch was set to ../ in the ansible.cfg that is created by .travis.yml, and the repo is called tripleo-collectd-ansible-role.

I can update this and check

Reply:
Looks like it was both? Latest Travis output has `[106] Role name tripleo-collectd-ansible-role does not match ``^[a-z][a-z0-9_]+$`` pattern`.